account_payment_order/tests/test_payment_order_outbound.py

```diff
@@ -416,13 +414,12 @@ def test_supplier_refund_reference(self):
         self.assertEqual("R1234", self.refund._get_payment_order_communication_direct())
 
         # The user add the outstanding payment to the invoice
-        invoice_line = self.invoice.line_ids.filtered(
+        self.invoice.line_ids.filtered(
```

Comment:
This is not doing anything right now, so please remove this code if not needed.

Reply:
Removed